Xamarin.Forms.Platform.Android/AppCompat/FormsAppCompatActivity.cs

```diff
@@ -23,6 +23,23 @@
 
 namespace Xamarin.Forms.Platform.Android
 {
+	[Flags]
+	public enum ActivationFlags : long
```

Comment:
is this the right place to put this? can we use this in Non app compact ?

Reply:
Yeah, that's a good point. These flags are all hints. So passing the same flags to all the different derivations of Activity would mean that derivation specific (compat vs not) would be ignored by some derivations. I'm inclined to unify them because I don't think they'll be that many derivations, my guess is many flags would overlap, and the documentation is simpler. But then again, if they don't overlap.. then maybe a nested class would per derivation would be better?

Reply:
Also, shouldn't everyone be using AppCompat anyway? The other ones are just legacy right? So I figure if someone is worried about perf then they'd need to upgrade to the latest greatest version anyway. And so these flags will only ever be passed into the AppCompat version and, God forbid, any future VeryAppCompat versions...